doc/product-variants-migration.md

```diff
@@ -0,0 +1,25 @@
+# Product variants migration
+
+If you used variant groups and/or inner variation types (with the paid extension InnerVariationBundle for the Enterprise Edition), or both together, the product migration is done in two steps. At first the products are migrated as they are not taking variants in account. Then in a second step, family variants, product models and product variants are created in the PIM 2.0 to reproduce the variants of the PIM 1.7.
+
+However, some bad use cases of variants are not supported anymore in 2.0. Here are the rules that must be respected in order to fully migrate the product variants :
```

Comment:
```
However , 2.0 PIM is stricter about the use of variations: it enforces to use them in a correct way. 
Therefore, some misconception in the use of the variation in 1.7 can prevent the migration of the variants in 2.0.

Here are the rules that must be respected in order to fully migrate the product variants :
```

Reply:
`the product variants:` -> no space before the `:`